inputFiles/poromechanicsFractures/PoroElastic_dfm_PEBICrack_smoke.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" ?>
+
+<Problem>
+
+  <Included>
+     <File name="./PoroElastic_dfm_PEBICrack_base.xml"/>
+  </Included>
+
+  <Solvers gravityVector="{0.0, 0.0, 0.0}">
+
+    <SinglePhasePoromechanicsConformingFractures
+      name="poroFractureSolver"
+      targetRegions="{ Region, Fault}" 
+      initialDt="1e-3"
+      flowSolverName="flowSolver"
+      solidSolverName="fractureMechSolver"      
+      logLevel="1">
+      <NonlinearSolverParameters
+        maxTimeStepCuts="5"
+        newtonTol="1e0"
+        newtonMaxIter="10" 
+        logLevel="2"
+        maxAllowedResidualNorm="1e+15"/>
+      <LinearSolverParameters
+        directParallel="0"/>
+
+    </SinglePhasePoromechanicsConformingFractures>
+
+    <SolidMechanicsLagrangeContact
+      name="fractureMechSolver"
+      stabilizationName="singlePhaseTPFA"
+      logLevel="1"
+      targetRegions="{ Region, Fault }"
+      timeIntegrationOption="QuasiStatic"
+      discretization="FE1"/>  
+
+    <SinglePhaseFVM 
+      name="flowSolver"
+      logLevel="1"
+      discretization="singlePhaseTPFA_stab"

```

Comment:
I would use `singlePhaseTPFA` here and `singlePhaseTPFA_stab` in the contact solver.